### ISSUE TITLE
Indent after numbers is not consistent with reference commonmark implementation

### ISSUE THREAD

**Issue by Wilfred** [atlassian/commonmark-java#85]
The following text:

```
1. foo


     this should not be code
```

Renders the last line as code, rather than text. I think this is a bug.

This does not match the reference commonmark implementation: http://spec.commonmark.org/dingus/

The majority of other implementations do not show this behaviour: http://johnmacfarlane.net/babelmark2/?normalize=1&text=1.+foo%0A%0A%0A++++bar

I believe the relevant part of the commonmark spec is [5.3 Lists](http://spec.commonmark.org/0.26/#lists), which says:

> A list is a sequence of one or more list items of the same type. The list items may be separated by any number of blank lines.

Note that this was confusingly worded in earlier versions of the spec: https://talk.commonmark.org/t/multiple-blank-lines-inside-a-list/2289

**Comment by robinst** [comment]
What version of commonmark-java are you using? Just tried with master, and the following string:

    "1. foo\n\n\n     this should not be code"

renders like this:

```
<ol>
<li>
<p>foo</p>
<p>this should not be code</p>
</li>
</ol>
```

This has been changed in commonmark-java 0.6.0 when we updated to spec 0.26, see release notes: https://github.com/atlassian/commonmark-java/releases/tag/commonmark-parent-0.6.0

**Comment by Wilfred** [comment]
Aha, excellent news!

I'm using commonmark-java on Bitbucket server v4.12.0. I don't suppose you know which commonmark-java versions are used in later versions of bitbucket server?

**Comment by robinst** [comment]
Just checked, 4.12.0 shipped with 0.5.1. Would you mind filing an issue for updating to a newer version here?: https://jira.atlassian.com/projects/BSERV

Thanks! :)
